### ISSUE TITLE
Make Google Analytics an opt-out option at build time

### ISSUE THREAD

**Issue by mvdan** [ushahidi/SMSSync#128]
It would be nice to have Google Analytics be optional at build time with gradle, using something like a noAnalytics flavour. Without it, we cannot build nor update your app on F-Droid, since it now requires non-free software (after #116).

I can provide a build.gradle patch if you want me to.

Regards.
